src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -30,6 +30,26 @@ public static class Constants
         /// </summary>
         public const string InProcDataCollectorsSettingName = "InProcDataCollectors";
 
+        /// <summary>
+        /// Coverlet data collector friendlyname
+        /// </summary>
+        public const string CoverletDataCollectorFriendlyName = "XPlat Code Coverage";
+
+        /// <summary>
+        /// Coverlet data collector assembly qualified name
+        /// </summary>
+        public const string CoverletDataCollectorAssemblyQualifiedName = "Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector.CoverletCoverageDataCollector, Microsoft.TestPlatform.Extensions.CoverletCoverageDataCollector, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
```

Comment:
value will be dummy for now. We will pick it later while validating inproc changes (from coverlet repo)